lib/controllers/european.controller.ts

```diff
@@ -75,16 +84,18 @@ export class EuropeanController extends SessionController {
           password: this.userConfig.password,
         },
         cookieJar,
+      }).catch(err => {
+        logger.debug(`Login failed: ${err}`);
```

Comment:
This won't expand the error and will likely output `[object Object]` cause I have not fixed the Winston logger yet.